pkgs/applications/virtualization/virtualbox/guest-additions/default.nix

```diff
@@ -63,8 +63,6 @@ stdenv.mkDerivation {
     do
         cd $i
         # Files within the guest additions ISO are using DOS line endings
```

Comment:
Probably would make sense to remove this comment as well, because it would otherwise be quite confusing.